BAZProjectC1/Services/ApiServiceRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ApiServiceRequest.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+public class ApiServiceRequest{
+    public static func getService<T: Codable>(urlService: String,structureType: T.Type, handler: @escaping(_ responseData: Any?) -> Void ){
```

Comment:
Es necesario que la respuesta genérica de un get conforme con `Codable`? Podría ser solo `Decodable`?

Te hice algunas sugerencias de estilo

```suggestion
    public static func getService<T: Codable>(urlService: String, structureType: T.Type, handler: @escaping(_ responseData: Any?) -> Void ) {
```